client/src/Header/Header.js

```diff
@@ -1,27 +1,40 @@
 import Container from 'react-bootstrap/Container';
 import Nav from 'react-bootstrap/Nav';
 import Navbar from 'react-bootstrap/Navbar';
-import {useState} from "react";
+import { useState } from "react";
 import Button from "react-bootstrap/Button";
+import {useNavigate} from "react-router-dom";
 
-export default function Header() {
+export default function Header({ isLoggedIn, setIsLoggedIn }) {
+    const onClick = () => {
+        setIsLoggedIn(isLoggedIn = false);
```

Comment:
setIsLoggedIn parameter should just be false.  Like: setIsLoggedIn(false)

Reply:
Done